packages/ui/src/components/Slider/components/DoubleSlider.tsx

```diff
@@ -137,8 +137,12 @@ export const DoubleSlider = ({
   const localId = useId()
   const finalId = id ?? localId
   const refSlider = useRef<HTMLInputElement>(null)
-  const [computedValue, setValues] = useState(value)
-  const [valueToShow, setValuesToShow] = useState<(number | null)[]>(value)
+  const safeValue =
+    value && typeof value === 'object' && value.length === 2
```

Comment:
```suggestion
    value &&  Array.isArray(value) && value.length === 2
```

Because {length: 2} is an objet and have length equals 2 ? :thinking: